src/QueueReceiver.Core/Services/PersonService.cs

```diff
@@ -27,27 +27,14 @@ public PersonService(IPersonRepository personRepository, IGraphService graphServ
 
             var adPerson = await _graphService.GetPersonByOid(userOid);
```

Comment:
Rename to smt like GetAdPersonByOid for readability? Was personally confused for a sec because of the similar method call for person on ln 22.